### ISSUE TITLE
KOTS Admin Console should support TLS

### ISSUE THREAD

**Issue by nathan-mellis** [replicatedhq/kots#645]
I would like to expose a link to the admin console through my application but it currently does not support being run using TLS. In order to expose it, I either need to require that the customer create a separate DNS entry and TLS certificate just for the admin console, or I need to port forward 8800 from my existing application ingress controller directly to the admin console, which only responds to HTTP. Since there are secrets and passwords, etc. that are directly interacted with through the admin console, we need to secure it with TLS. And asking our customers to set up a separate domain and TLS certificate is also not a good solution.

**Comment by nathan-mellis** [comment]
Note: This is also related to the other issue I filed in #646. My preference would be for this to be solved using that path (e.g. expose it as a subpath under the main application DNS and TLS).

**Comment by marccampbell** [comment]
@nathan-mellis agreed that the better solution here is to support a subpath and let your application handle the TLS termination. 

KOTS would support deploying a LoadBalancer and having TLS terminated outside or in an ingress controller, but it's not trivial and not built in today.

We are prioritizing #646 first. Let's leave this one open for a bit if 646 solves your immediate needs?

**Comment by ddvdozuki** [comment]
Any update to this? I'd love subpath support but this would work as well. Just really need TLS on the admin dashboard since we allow clients to access the dash.
